deploy/kubernetes/metricbeat-kubernetes.yaml

```diff
@@ -151,7 +151,7 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
       - name: metricbeat
-        image: docker.elastic.co/beats/metricbeat:7.17.15
+        image: docker.elastic.co/beats/metricbeat:
```

Comment:
Should the image version remain empty? does not seem right?

Reply:
@gizas this was because of the files generation.
Do you know why the make command is generated them without a version?

Reply:
This happened in the 8.11 backport too, but didn't happen on main.